### PR TITLE
Inline-button-create: Add href

### PR DESCRIPTION
**Pull request by JesmoDev**
Adds href and target properties to inline button create
Also fixes the placement of the button so it is no longer offset to the side.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

- [x] If my change requires a change to the documentation, I have updated the documentation in this pull request.
- [x] I have read the **[CONTRIBUTING](<(https://github.com/umbraco/Umbraco.UI/blob/v1/contrib/docs/CONTRIBUTING.md)>)** document.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: packages/uui-button-inline-create/lib/uui-button-inline-create.element.ts [hunk 1]
```diff
@@ -37,8 +38,26 @@ export class UUIButtonInlineCreateElement extends LitElement {
...
+   * @type {string}
+   * @attr
+   * @default undefined
+   */
+  @property({ type: String })
+  public target?: '_blank' | '_parent' | '_self' | '_top';
+
   private _onMouseMove(e: MouseEvent) {
-    this._position = this.vertical ? e.offsetY : e.offsetX;
+    this._position = (this.vertical ? e.offsetY : e.offsetX) + 5;
```
Comment (iOvergaard) [inline packages/uui-button-inline-create/lib/uui-button-inline-create.element.ts:60]: Why 5 exactly? 😎

**Comment by iOvergaard** [comment]
See my comment on ref-node also: https://github.com/umbraco/Umbraco.UI/pull/715#discussion_r1463313699

File: packages/uui-button-inline-create/lib/uui-button-inline-create.element.ts (hunk 1 above)
Comment (nielslyngsoe) [reply packages/uui-button-inline-create/lib/uui-button-inline-create.element.ts:60]: Yeah, this is counter offsetting what was offset via CSS.

I have removed this, and adjusted the CSS as the cursor was a bit too off. Now it looks like this, moving the button a bit off so you can see the plus, but still it feels like the mouse is on top. (Before is was just on the edge)
IMG_0091

**Review by nielslyngsoe — APPROVED** [review]
good

**Pull request merged**